samples/bluetooth/mesh/dfu/distributor/prj.conf

```diff
@@ -103,3 +103,7 @@ CONFIG_SEGGER_RTT_BUFFER_SIZE_DOWN=100
 # MCUmgr UART transport
 CONFIG_BASE64=y
 CONFIG_MCUMGR_TRANSPORT_UART=y
+
+# This option may not be enabled automatically for some boards. Enabling it explicitly to avoid
+# missing configuration.
+CONFIG_USE_SEGGER_RTT=y
```

Comment:
Should CONFIG_SHELL_BACKEND_SERIAL be set to y?

Reply:
no, it is for UART.